tests/inferences/test_renyi_divergence.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import edward as ed
+import numpy as np
+import tensorflow as tf
+
+from edward.models import Bernoulli, Normal
+from edward.inferences.renyi_divergence import RenyiDivergence
```

Comment:
We should check the import also works by using `ed.RenyiDivergence` in the test.

Reply:
Got some issue with this but it should be working now.